doc/radosgw/STS.rst

```diff
@@ -90,17 +90,11 @@ RGW now supports Session tags that can be passed in the web token to AssumeRoleW
 STS Configuration
 =================
 
-The following configurable options have to be added for STS integration::
+.. confval:: rgw_sts_key
```

Comment:
Shouldn’t there be a sentence describing the options?

Reply:
is the usage of confval redirecting us to read the description from rgw.yaml?

Reply:
you can see how this renders in https://ceph--63360.org.readthedocs.build/en/63360/radosgw/STS/#sts-configuration

Reply:
Fair enough, I would still lead with a contextual sentence like:

The following central config options must be set for STS integration: